### PR TITLE
Show thumbnails for on the search results view

### LINKED ISSUE #173: Show Thumbnails on the search results view
Revert this:
https://github.com/projecthydra-labs/curation_concerns/commit/5cb4ccd85a5d0732c3cbdeda14fd7a5614011e10#diff-aaf6ef0ebf5b3b4d403c46733670b421L2

### PR DESCRIPTION
**Pull request by jcoyne**
Also adds a thumbnail for audio files
Fixes #173

### CODE REVIEW

File: app/views/catalog/_index_default.html.erb [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <div class="row">
-  <%#= render_document_partial document, 'thumbnail', document_counter: generic_file_counter %>
+  <%= render 'thumbnail_default', document: document, document_counter: document_counter %>
```
Comment (cam156) [inline app/views/catalog/_index_default.html.erb]: Is thumbnail_default a blacklight view?  This seems a bit mysterious.

File: app/views/curation_concerns/generic_files/_generic_file.html.erb [hunk 2]
```diff
@@ -1,5 +1,5 @@
 <tr class="<%= dom_class(generic_file) %> attributes">
-  <td>
+  <td class="thumbnail">
     <%= render_thumbnail_tag generic_file %>
```
Comment (cam156) [inline app/views/curation_concerns/generic_files/_generic_file.html.erb]: @jcoyne Just curios why we are using two separate thumbnail renders.  Would this helper do the right thing in index_default @ line 2?

File: app/views/catalog/_index_default.html.erb (hunk 1 above)
Comment (terrellt) [inline app/views/catalog/_index_default.html.erb]: It is. https://github.com/projectblacklight/blacklight/blob/master/app/views/catalog/_thumbnail_default.html.erb

**Comment by cam156** [comment]
@jcoyne :clap: The concept of sending a url to solr in nice!

File: app/views/curation_concerns/generic_files/_generic_file.html.erb (hunk 2 above)
Comment (jcoyne) [inline app/views/curation_concerns/generic_files/_generic_file.html.erb]: The other render is a partial specifically for search results

**Comment by cam156** [comment]
@jcoyne If there was a green button I would push it.  Unfortunately it needs to be rebased.

**Comment by jcoyne** [comment]
@cam156 I made a green button for you.

**Pull request merged**